source/get-started/virtual-machine-install/kvm.rst

```diff
@@ -102,42 +89,27 @@ Download and launch the virtual machine
 
```

Comment:
Starting on Line 66, above, please remove sentences that start with "For example, the |CL| ..." 
I'm sure other developers might appreciate that on the one hand. On the other, I don't want our docs  to support other distro's docs --especially with supporting files. If you want to call out that example, please post those sentences in the Forum. You could also add those lines to an announcement of this much improved `kvm` guide there too.

Reply:
This was the original content and unmodified as part of the changes in this PR. I agree with you and have made the requested changes.

Reply:
Thank you.